api/pkg/s3/bucketget.go

```diff
@@ -69,10 +70,6 @@ func checkObjKeyFilter(val string) (string, error) {
 }
 
 func (s *APIService) BucketGet(request *restful.Request, response *restful.Response) {
-	if !policy.Authorize(request, response, "bucket:get") {
-		return
-	}
-
 	limit, offset, err := common.GetPaginationParam(request)
```

Comment:
Why delete policy checking here

Reply:
Moved to RouteBucketGet which is located in routebucketget.go, more than one APIs will be routed through  RouteBucketGet.